librz/analysis/arch/arm/arm_il32.c

```diff
@@ -482,7 +652,9 @@ static RzILOpEffect *add_sub(cs_insn *insn, bool is_thumb) {
 		// alias for adr
 		return adr(insn, is_thumb);
 	}
-	bool is_sub = insn->id == ARM_INS_SUB || insn->id == ARM_INS_SUBW || insn->id == ARM_INS_RSB || insn->id == ARM_INS_RSC || insn->id == ARM_INS_SBC;
+	bool is_sub =
```

Comment:
Maybe move those long ORs into their own inline function. Looks a little nicer. Also above and below.

Reply:
Agree on that

Reply:
ok

Reply:
And this one